docs/modules/record/README.md

```diff
@@ -1,85 +1,32 @@
-# Record User Guide
+# iriscli record
 
-## Basic Function Description
+## Description
 
-1. On-Chain record for the text data
-2. On-Chain record for the text file (TODO)
-3. On-Chain governance for the record params (TODO)
+Record allows you to submit, query and download your records on chain.
 
-## Interaction Process
-
-### Record process
-
-1. Any users can initiate a record request. It will cost you some tokens. If there’s no record of the data on the existing chains, the request will be completed successfully and the relevant metadata will be recorded onchain. And you will be returned a record ID to confirm your ownership of the data.
-2. If any others initiate a record request for the same data, the request will be directly rejected and it will hint that the relevant record data has already existed.
-3. Any users can search/download onchain based on the record ID.
-4. At present, the maximum amount of stored data at most 1K Bytes. In the future, the dynamic adjustment of parameters will be implemented in conjunction with the governance module.
```

Comment:
the maximum amount of stored data is 1K Bytes.

Reply:
Got it.